app/src/editor/canvas/components/Resizable/index.jsx

```diff
@@ -112,6 +116,7 @@ const Resizable = ({
     const value = useMemo(() => ({
         ...size,
         enabled: true,
+        toggleHandle: setShowHandle,
```

Comment:
not sure if "toggle" is the right verb since it doesn't do any toggling, it just sets the value, right?

Reply:
Removed custom name and went directly with `setShowHandle`. I suck at naming. xD